tests/markdown_splitCjkText/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1,6 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`cjk.md - markdown-verify 1`] = `
+exports[`cj.md - markdown-verify 1`] = `
```

Comment:
What do you think about renaming these to `chinese-japanese.md` and `korean.md` so it’s clearer at first glance what they’re testing?

Or are `cj` and `k` common abbreviations?

Reply:
I've seen CJK, but never CJ and K as separate abbreviations